css/header.css

```diff
@@ -0,0 +1,4 @@
+.header{
+    border-bottom: none !important;
+    box-shadow: rgba(78, 86, 88, 0.1) 0px 10px 15px -3px, rgba(186, 223, 238, 0.05) 0px 4px 6px -2px !important; 
```

Comment:
why important here

Reply:
I have used the components from my component library. I wanted to override the existing CSS. Without `!important` it wasn't overriding the styles. So, I could either use inline style or use `!important` keyword.

Reply:
fixed the issue.